packages/terra-data-grid/src/FlowsheetDataGrid.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import DataGrid from './DataGrid';
+import rowShape from './proptypes/rowShape';
+import { columnShape } from './proptypes/columnShape';
+import styles from './FlowsheetDataGrid.module.scss';
```

Comment:
Nit: but I would prefer grouping imports into 3rd party, terra and then internal imports for better readability.
```suggestion
import React from 'react';
import PropTypes from 'prop-types';
import classNames from 'classnames/bind';

import DataGrid from './DataGrid';
import rowShape from './proptypes/rowShape';
import { columnShape } from './proptypes/columnShape';
import styles from './FlowsheetDataGrid.module.scss';
```